bindings/java/src/org/sleuthkit/datamodel/DataSource.java

```diff
@@ -83,4 +95,40 @@
 	 * @throws TskCoreException Thrown if the data can not be read
 	 */
 	String getAcquisitionDetails() throws TskCoreException;
+
+	/**
+	 * Gets the acquisition settings field from the case database.
+	 *
+	 * @return The acquisition settings
+	 *
+	 * @throws TskCoreException Thrown if the data can not be read
+	 */
+	String getAcquisitionSettings() throws TskCoreException;
+
+	/**
+	 * Gets the module name field from the case database.
+	 *
+	 * @return The module name
+	 *
+	 * @throws TskCoreException Thrown if the data can not be read
+	 */
+	String getModuleName() throws TskCoreException;
```

Comment:
If module name and version are part of the acquisition settings, why have separate getter methods for the acquisition settings, module name and module version? We seem to be overloading the term "acquisition settings."

We could define an AcquistionSettings class and return an instance from a single getAcquisitionSettings method. If we pass an AcquistionSettings object to setAcquistionSettings as well, it would future-proof us if there is any possibility that AcquisitionSettings will expand in the future. 

Or maybe the acquisition settings string is actually module configuration?

Or maybe module name and module version are not acquisition settings at all?

Reply:
@rcordovano At this stage, I am ok with having the module version and module name be part of the AcquistionSettings string (since I am putting a JSON string into settings) and remove the dedicated apis. 

This will eliminate the need for a new class and the settings can be whatever the modules need. 

Let me know if you prefer a new class instead.

Reply:
@jayaramcs, I am good with either solution. I'd like to hear what @bcarrier prefers, since his design called for separate columns in the case database.